manifests/init.pp

```diff
@@ -5,6 +5,7 @@
 class stackdriver (
   $manage_repo = true,
   $baseurl     = undef,
+  $gpgcheck    = true,
```

Comment:
Thanks for the contribution! The new parameter needs to be documented in the README and be added to the spec tests.